src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -345,8 +345,8 @@ def add_ci(
 
     # Call the RP
     return _call_rp_configure_cicd(
-        name,
-        resource_group_name,
+        target_name,
+        target_resource_group,
```

Comment:
@tjprescott @derekbekoe  This is what our command looks like: az container release create -n targetName -g resourcegroup .....
These parameters are not used to create a release with name 'targetName' but actually point to an ACS instance of the same name. This would have confused a user expecting the release to be created with these parameters as they are pointing to the target ACS instance where the release will be deployed.